source/2020-01-17-the-ember-times-issue-131.md

```diff
@@ -36,13 +37,13 @@ Share your thoughts, feedback, and requests on the [New Test Waiters RFC](https:
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Optimizing your app with Ember Data 📈](https://runspired.com/2019/12/15/optimizing-your-app-with-ember-data/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Chris Thoburn (@runspired)](https://github.com/runspired) is blogging about how to build and optimize an app with Ember Data. You'll start by building an app, Listicle. Listicle starts as a small app shell with rich content lists, but balloons in size over time. Build times slow to a crawl as a result. 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+At the beginning of the series, Listicle builds and renders in greater than 5 seconds. But as the posts progress, you'll end with an app with builds and renders in **less than 1 second** by optimizing only the app's data management! Then for fun, you'll optimize the render. You can check out [Part 1](https://runspired.com/2019/12/15/optimizing-your-app-with-ember-data/) and [Part 2](https://runspired.com/2019/12/18/optimizing-your-app-with-ember-data-part-2/) now, with more posts to come! And you can follow along with the code for the series by watching the [listicle repository](https://github.com/runspired/listicle).
```

Comment:
I think it may be nice to break the paragraph into two (change is optional and non-blocking).
```suggestion
At the beginning of the series, Listicle builds and renders in greater than 5 seconds. But as the posts progress, you'll end with an app with builds and renders in **less than 1 second** by optimizing only the app's data management! Then for fun, you'll optimize the render.

You can check out [Part 1](https://runspired.com/2019/12/15/optimizing-your-app-with-ember-data/) and [Part 2](https://runspired.com/2019/12/18/optimizing-your-app-with-ember-data-part-2/) now, with more posts to come! And you can follow along with the code for the series by watching the [listicle repository](https://github.com/runspired/listicle).
```

Reply:
sgtm, thank you!

Reply:
committed manually, guess you have to commit suggestions first